pkgs/applications/misc/xygrib/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, bzip2, qt5, libnova, proj, libpng, openjpeg, makeWrapper } :
+
+stdenv.mkDerivation rec {
+  version = "1.2.6.1";
+  name = "xygrib-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opengribs";
+    repo = "XyGrib";
+    rev = "v${version}";
+    sha256 = "0xzsm8pr0zjk3f8j880fg5n82jyxn8xf1330qmmq1fqv7rsrg9ia";
+   };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ bzip2 qt5.full libnova proj openjpeg libpng makeWrapper ];
+  cmakeFlags="-DOPENJPEG_INCLUDE_DIR=${openjpeg.dev}/include/openjpeg-2.3";
+  postInstall = ''makeWrapper $out/XyGrib/XyGrib $out/bin/XyGrib'';
+
+  meta = with stdenv.lib; {
+    homepage = https://opengribs.org;
+    description = "Weather Forecast Visualization";
+    longDescription = ''XyGrib is a leading opensource weather visualization package that now interacts with OpenGribs's Grib server providing a choice of global and large area atmospheric and wave models.'';
```

Comment:
Please split it into multiple lines, you're already using the quotes that support that :)